examples/sql/sql_json_example.py

```diff
@@ -0,0 +1,34 @@
+import hazelcast
```

Comment:
How about enabling logging for the example? It increases the LOC but in case there is a problem, that would help the user to spot the problem.

Reply:
I just checked the other code samples, and apparently, we haven't enabled logging in any of them. But, I agree with you and I believe it would definitely be useful. So, if it is okay for you, I will create an issue for it, and will enable logging for all code samples in a later PR

Reply:
Sure, thanks!